### DISCUSSION TITLE
knowledge base

### DISCUSSION THREAD

**Discussion by sahilsharma-tmpl** [infiniflow/ragflow#3500] [Q&A] [answered]
i want to know if there is any way to stop the chatbot from answering questions from datasets not included in its setting. for example if i ask  chatbot a question related to a dataset which is present in ragflow but is not associated with that chatbot , it still gives answer for that question which it should not. does any one have a solution

**Comment by KevinHuSh** [comment, marked as answer]
It's a bug and fixed already.

**Reply by sahilsharma-tmpl** [reply]
in the latest update?

**Reply by sahilsharma-tmpl** [reply]
i'm using the latest version but still facing the above issue !

**Comment by KevinHuSh** [comment]
Pull the dev version of docker image.

**Reply by sahilsharma-tmpl** [reply]
its done thank you
